src/main/java/uk/gov/ons/ssdc/supporttool/model/messaging/dto/EventTypeDTO.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.ons.ssdc.supporttool.model.messaging.dto;
+
+public enum EventTypeDTO {
```

Comment:
Most of these are unused... consider cutting down to just the ones which are used?

Reply:
Good spot, bad copypasta sorry